src/fake-timers-src.js

```diff
@@ -143,6 +155,30 @@ function withGlobal(_global) {
         throw new TypeError("now should be milliseconds since UNIX epoch");
     }
 
+    function getInfiniteLoopError(loopLimit, job) {
+        var infiniteLoopError = new Error(
+            "Aborting after running " +
+                loopLimit +
+                " timers, assuming an infinite loop!"
+        );
+
+        var sliceSize = job.type ? 4 : 3;
```

Comment:
It would be ideal to calculate this by looking for ourselves in the stack trace rather than setting prefixed sizes. This can fail based on adding new jobs.